- adding artwork working folder for some new species icons

st-pa-fo opened the pull request:
namely of Nightsiders, Sleepers, Khakturian, Celestephyte, Thenian. they all have their own named threads in the story subforum except nightsiders who reside in the species master list thread.

st-pa-fo committed:
- adding zipped artwork working folder for species icons of Nightsiders, Sleepers, Khakturian, Celestephyte, Thenian

st-pa-fo commented:
i was planning to go through the licences of the images used and add them to some readme or license file when the nightsiders-pullrequest gets merged, posted links or noted sources as accompanying text files in most cases

TheSilentOne1 commented:
There is no need to compress the artwork; it also makes reviewing more difficult. Looking at the zip I think you should clean up the files a bit before commiting them to the asset repository, if at all - for small icons there is little use for source files.

st-pa-fo committed:
- removing zip again
- adding uncompressed and only relevant artworks files with accompanying licencing information

st-pa-fo commented:
either i havent looked hard enough, or the models on which the current species icons are based are just not there. where would i find those?

TheSilentOne1 commented:
The art repository was introduced somewhere down the road, and many artists didn't leave their source files, so it is far from complete.

Folders: icons/species for species icons, and concept_art/species for the lichens. Your name / nick can go into the .md files, which you should add instead of .txt.
Remove the face from the cactus, and maybe also no spikes or fewer more distinctive ones.
The penguin doesn't work on small sizes, I  would suggest a different shape.

TheSilentOne1 commented:
Also, have you made sure the license of the art you obtained is compatible with the one used by freeorion (Creative Commons Attribution-ShareAlike 3.0 Unported)?

st-pa-fo commented:
was all marked as public domain, i hope that is compatible.

> I would suggest a different shape.

ok i'll look for something different.

> Remove the face from the cactus

aye

and .. as i said earlier, these icons were ment as placeholders only, until something better comes along. i'll do another round of editing, but there comes a point when i would think it pointless to put much more energy into placeholder images to be replaced by something better anyway when an inclined artist comes along ..

st-pa-fo committed:
- incorporating feedback: new khakturian without face and bigger spikes, and something completely different for the sleepers

st-pa-fo commented:
<img width="740" alt="updated khakturian" src="https://user-images.githubusercontent.com/37298400/150696115-d5a57193-6280-4108-b9a9-21dc8a036e51.png">
<img width="752" alt="updated sleepers" src="https://user-images.githubusercontent.com/37298400/150696121-2ab57dbc-f15d-40ed-a872-7f0b8cdb5ea8.png">

TheSilentOne1 commented:
Kakturian much better I think, nice job.
How about quadruped-05 for Sleepers?
![quadruped-05](https://user-images.githubusercontent.com/12985960/150789659-c71e509e-f0f2-41ff-9018-77c354b17e44.png)
Celestophyte would probably look better without the "eyes".

st-pa-fo commented:
aye, i'll remove the celestephyte's eyes and try to make it look better at smallest size in census and small size on planets, i think the eyes only were visible in the big one on pedia pages.

and i'll try that cute little fuzzball image but i'll leave the icons in the artwork repository anyway. when i'm done here, i'll push them in the nightsider pr too, which has to be rebased on master again.

st-pa-fo committed:
- incorporating more feedback: new celestephyte without face and better visible on planets

st-pa-fo commented:
<img width="825" alt="celestephyte updated" src="https://user-images.githubusercontent.com/37298400/150860878-7a5a037d-4cc4-4ec7-91b5-267da9d0ffa8.png">

there.

st-pa-fo commented:
@Vezzra --> seems like the models for the original preexisting lowres icons are simply not there, might have given it a shot otherwise .. but how about lets talk internationalization instead ;?

st-pa-fo committed:
- add khakturian derelict trooper sitrep icon concept stuff

Vezzra commented:
@st-pa-fo,

> seems like the models for the original preexisting lowres icons are simply not there

Yeah, unfortunately the assets repo has only been introduced a couple of years ago. Before then, the models (or other source files) which were used to generate the various images haven't been tracked, and a large part of them is lost. So there are a lot of images for which we do not have the models/source files anymore.

> but how about lets talk internationalization instead ;?

Not sure what you mean...?

st-pa-fo commented:
> Not sure what you mean...?

see also https://www.freeorion.org/forum/viewtopic.php?p=109421#p109421 where i tried to revive discussion about localization workflow. i'd like to do some translating. but i'd like to do that with omegaT. and there is lots still to be done to the codebase before the japanese translation that is underway atm by ganbaranaiyo has a chance of achieving a really japanese-ish look and feel.

and this had not a lot to do with this pr here anyway but answers the question.

all those preexisting lowres species icons currently in the project but not in use for any species - i should have been able to discover them earlier, before spending days with gimp to fumble around with colour contrast layers sharping edges rescale and so on .. for future species i'll just use up those preexisting icons if noone has any prior claims there

Vezzra commented:
> and this had not a lot to do with this pr here anyway but answers the question.

Ah, ok. Well, this was what I was confused about - what internationalization has to do with this PR... :wink:
